project/src/main/webapp/components/user-home/folder-component.js

```diff
@@ -0,0 +1,9 @@
+import {html, LitElement} from 'https://unpkg.com/@polymer/lit-element/lit-element.js?module';
+import {DocsComponent} from './docs-component.js';
+
+export class FolderComponent extends DocsComponent {
```

Comment:
If this class is the same thing as the parent class, is there any reason you have to create a new child class instead of reusing the parent?

Reply:
I assign its attribute when its used in user-home.js; I structured it like this for more clarity on what is what in user-home. ie, it's more descriptive to have a <folder-component> and <my-docs-component> than two <docs-component> with different attributes. 

This class is pretty useless right now, so I'm not opposed to just deleting it as well.

Reply:
For clarify I am fine with having a separate class, in case we want to have different styling or other distinguishment in the future.